contrib/exporters/secadvisor/mod/account_file.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+func makeTimestamp() int64 {
+	return time.Now().UnixNano() / int64(time.Millisecond)
+}
+
+type accountFile struct {
+	filename string
+}
+
+type accountRecord struct {
```

Comment:
add field called 'version' having some known version number and check the number in read/write ... this will be useful as we can easily assume that this file will be changed due to future req. from billing / accounting

Reply:
I can add a version number and for now just abort if the version of the file if not identical to version of software (and later we can consider migrating old format to new)